src/Benchmarks/Benchmarks.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net451</TargetFrameworks>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
There is no conversation to move forward. We we're debating common.props vs dependencies.props and decided dependencies.props makes more sense. It doesn't make sense to me because it's not a package version like the other things in `dependencies.props`. You can say it is, but it really isn't.